roles/comanage-registry/templates/security.seed.j2

```diff
@@ -0,0 +1 @@
+{{ vault_comanage_security_seed }}
```

Comment:
The use of vault should not be required by the template. We need to change this to comanage_security_seed and add the definition in defaults/main.yml (commented out) explaining how this can be generated or linking to the relevant documentation